Microwave.Test.Unit/BeeperTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microwave.Classes.Boundary;
+using Microwave.Classes.Interfaces;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Microwave.Test.Unit;
+
+[TestFixture]
+public class BeeperTest
+{
+	private IOutput output;
+	private Beeper uut;
+
+	[SetUp]
+	public void Setup()
+	{
+		output = Substitute.For<IOutput>();
+		uut = new Beeper(output);
+	}
+
+	[Test]
+	public void Start_BeepOutputThreeTimes()
+	{
+		uut.Start();
+		output.Received(3).OutputLine("* B E E P *");
```

Comment:
Brug Arg.Is til at tjekke om OutputLine får korrent  argument

Reply:
Fixet i https://github.com/nikolajLaerke1/MicrowaveHandin3/pull/2/commits/2be9c8aeb9bbed27bf7fb81089bebabbc09a9295